packages/integration-tests/src/test-e2e/sandbox/sandbox.test.template.ts

```diff
@@ -86,8 +86,12 @@ export const defineSandboxTest = (testProjectCreator: TestProjectCreator) => {
           if (updates.length > 0) {
             // retry hotswapping resources if deployment time is higher than the threshold
             await runWithRetry(
-              async () => {
-                // keeping initial deployment in retry loop to reset app state for each hotswap to be a non no-op
+              async (attempt) => {
+                if (attempt > 1) {
```

Comment:
nit: we could skip this if. reset of fresh project is no-op.
this is more explicit though so feel free to disregard.

Reply:
That's true but I preferred being more explicit here so the project doesn't reset when it doesn't need to on first attempt of hotswap.